spec/api_spec.rb

```diff
@@ -50,7 +60,7 @@
     let(:collection) { {'123456' => interviewer_record_in_db} }
     it "returns interviewer with key=:id" do
       get "/api/v1/interviewer/#{key}"
-      expect(json_body).to eql({name: 'Jichao', email: 'jichao@thoughtworks.com', key: '123456'})
+      expect(json_body).to eql({name: 'Jichao', email: 'jichao@thoughtworks.com', key: '123456', experience:4, language: 'ruby', work_from: '2011'})
```

Comment:
Line is too long. [148/80]<br>Redundant curly braces around a hash parameter.<br>Space missing after colon.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.